demo/universal/js/ReactServerDOMEsbuild.re

```diff
@@ -1,22 +1,21 @@
-type callServer('arg, 'result) =
-  (string, list('arg)) => Js.Promise.t('result);
+type arg;
+type callServer = (string, list(arg)) => Js.Promise.t(React.element);
```

Comment:
The `callServer` is also used for server functions that do not return a React element.
This is the callServer type on React: https://github.com/facebook/react/blob/3d14fcf03f4e296d21b52b362f5adefd9e366375/packages/react-client/src/ReactFlightReplyClient.js#L54